HealthcareSystem/Backend/Service/Encryption/EncryptionService.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Backend.Service.Encryption
+{
+    public class EncryptionService
+    {
+        public EncryptionService() { 
+        }
+        public string DecryptString(string encryptedJmbg)
+        {
+            char[] arr;
+            string result = "";
+            arr = encryptedJmbg.ToCharArray(0, 13);
+
+            foreach (char c in arr)
+            {
+                if (c.ToString() == "A")
+                {
+                    result += "0";
+                }
```

Comment:
I am wondering why **'c'** and **'arr'**.
(I do not say it is incorrect, but I am confused)

Reply:
arr is abbreviated from array
c is abbreviated from character